conbench/tests/api/_expected_docs.py

```diff
@@ -1132,7 +1139,7 @@
                         "type": "object",
                     },
                     "error_type": {
-                        "description": "Run's error type. Possible values: none, catastrophic, partial. \n                    None = all attempted benchmarks are good. \n                    Catastrophic =no benchmarks completed successfully. \n                    Partial = some benchmarks completed, some failed",
+                        "description": "Run's error type. Possible values: none, catastrophic, partial.\n                    None = all attempted benchmarks are good.\n                    Catastrophic =no benchmarks completed successfully.\n                    Partial = some benchmarks completed, some failed",
```

Comment:
I don't see descriptions for other fields. 
The way to update this file with new doc strings from marshmallow is to run tests locally (or just one test) which will update this expected doc.

Reply:
Ok, finally got all the pieces in place to run the tests *not* in docker so that I get the updated file along with it. I wonder if it would be possible to have the docker-based tests update the file too?